### ISSUE TITLE
Issues in V0.46.3

### ISSUE THREAD

**Issue by KururinX** [DS-Homebrew/nds-bootstrap#1274]
Confirmed in v0.46.3
● DS organize Freeze trying ...
● Some commercial ROMs that started on the DSi do not start on the DSi LL
Report that these issues have been resolved by 31d05dbf and v0.47.0 and 806a594f.

* Not all of the latter have been confirmed.
** DSi has not confirmed that the problem has been resolved.

**Comment by RocketRobz** [comment]
The reason as to why homebrew wasn't working in v0.46.3, is because a file was left out by mistake.
The file has been added in v0.47.0, so your homebrew issues should no longer occur. If they still do occur though, I have no idea why.
As for commercial ROMs, the bug should also be fixed in v0.47.0.
I've gotten the same issue on my side, and was able to fix that bug as well. I've tested other games, and can confirm that the bug is indeed fixed. Once again, if the bug still occurs, I have no idea why.

Make sure both consoles are on the latest version.
